src/operations/search/query/r4.js

```diff
@@ -122,7 +122,23 @@ module.exports.buildR4SearchQuery = (resource_name, args) => {
         columns.add('meta.lastUpdated');
     }
 
-    for (const [field, filterObj] of Object.entries(customQueries)) {
+    for (const [resourceType, filterObj] of Object.entries(customScalarQueries)) {
+        if (resourceType === resource_name) {
+            for (const [property, propertyObj] of Object.entries(filterObj)) {
+                if (args[`${property}`]) {
+                    // console.log(property + ' = ' + propertyObj.type + ' , ' + propertyObj.field);
+                    switch (propertyObj.type) {
+                        case 'instant':
```

Comment:
Will the switch case here grow? If it does, should we create a separate method for it to make it easier to test.

Reply:
we can refactor in the future when it grows